1.x/docs/Installing-sbt-on-Mac.html

```diff
@@ -46,7 +46,7 @@ <h2 id="Installing+sbt+on+macOS">Installing sbt on macOS<a href="#Installing+sbt
 </p><p>Or use <a href="https://sdkman.io/">SDKMAN!</a>:
 </p><pre><code class="">$ sdk list java
 $ sdk install java 11.0.4.hs-adpt
-</code></pre><h3 id="Installing+from+a+universal+package">Installing from a universal package<a href="#Installing+from+a+universal+package" class="header-link"><span class="header-link-content">&nbsp;</span></a></h3><p>Download <a href="https://github.com/sbt/sbt/releases/download/sbt-1.4.4.zip">ZIP</a> or <a href="https://github.com/sbt/sbt/releases/download/sbt-1.4.4.tgz">TGZ</a> package, and expand it.
+</code></pre><h3 id="Installing+from+a+universal+package">Installing from a universal package<a href="#Installing+from+a+universal+package" class="header-link"><span class="header-link-content">&nbsp;</span></a></h3><p>Download <a href="https://github.com/sbt/sbt/releases/download/v1.4.4/sbt-1.4.4.zip">ZIP</a> or <a href="https://github.com/sbt/sbt/releases/download/v1.4.4/sbt-1.4.4.tgz">TGZ</a> package, and expand it.
```

Comment:
Thanks for the contribution. This HTML was generated from markdown.
Please fix here
https://github.com/sbt/website/blob/develop/src/reference/00-Getting-Started/01-Setup/a.md
https://github.com/sbt/website/search?p=1&q=sbt_native_package_base